src/themes/elife/test/fixtures/referencesData.fixture.ts

```diff
@@ -0,0 +1,62 @@
+import { create } from '../../../../util'
+
+/* Using this canned data is not great as it's a snapshot in time of how the reference comes over from
+   Encoda, which may change. A possible future improvement could be to read the file
+     /src/examples/articleReplication.html
+   and use the reference list within as the data source for the unformatted references.
+   This would make sure that the tests are run against the current markup and so stay
+   relevant. It would also run the tests against different types of reference, making them
+   more useful at identifying gaps in the formatting, and any regressions that might
+   creep in.
+ */
+// Reference 5 from articleReplication.html example.
+const unformattedReference =
+  '<li itemscope="" itemtype="http://schema.org/Article" itemprop="citation" id="bib5">\n' +
+  '        <ol data-itemprop="authors">\n' +
```

Comment:
Does this need formatting properly so it is indented nicely?

Reply:
This is how my IDE understands that Prettier wants it formatted, and the checks pass, so from that I'm confident it's per the formatting requirements of this repo.